docs/plans/2026-04-25-component-templates-plan.md

```
@@ -0,0 +1,1473 @@
+Expected: `syntax ok`.
+
+### Task A2: Wire `ws component` into the main router
+
+**Files:**
+- Modify: `D:/Dev/GitWS/yggdrasil/scripts/ws`
+
+- [ ] **Step 1: Add help-block entries**
+
+In `D:/Dev/GitWS/yggdrasil/scripts/ws`, locate the `# Commands:` block at the top of the file (around line 8-32). Find the existing line:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)